main/ertp/guide/amount-math.md

```diff
@@ -10,111 +10,88 @@ uses `amountMath` methods for all these operations.
 `amountMath` methods also check their arguments' `brand`, 
```

Comment:
in line 8 above, let's say: ERTP uses the `amountMath` library

Reply:
Line 10 is no longer true, unless the optional `brand` is used in some methods. For instance, `amountMath.make` does not error if a "wrong brand" is used. In fact, it has no concept of a wrong brand.